### PR TITLE
updating README and tests

### PR DESCRIPTION
**Pull request by pennywisdom**
Updating the README and tests to give a little more information and some examples. Tests now check for working http response rather than just displaying the logs.

### CODE REVIEW

File: caddy/test.ps1 [hunk 1]
```diff
@@ -1,3 +1,10 @@
 docker run --name caddy -d -p 8080:80 -v "$(pwd)\conf:C:\caddy\conf" -v "$(pwd)\www:C:\wwwroot" caddy:0.10
 Start-Sleep -Seconds 5
+$req = Invoke-WebRequest http://$(docker inspect -f '{{ .NetworkSettings.Networks.nat.IPAddress }}' caddy) -UseBasicParsing
```
Comment (StefanScherer) [inline caddy/test.ps1]: That's awesome that you added a real test! I also thought about it.

**Comment by pennywisdom** [comment]
@StefanScherer looks like the appveyor tests timed out getting the latest windows images..

**Comment by StefanScherer** [comment]
I'm restarting build.

**Comment by StefanScherer** [comment]
This time it worked. Thanks again for the test.

**Pull request merged**